source/deployment/deployNetworks.ts

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env node
+
+import { deployContracts } from "./deployContracts"
+import { Configuration } from '../libraries/Configuration';
+
+export async function deployToNetworks(networks: Array<string>) {
+    for(let network of networks) {
+        const configuration = Configuration.network(network);
+        await deployContracts(configuration);
+    }
+}
+
+if (require.main === module) {
+    const networks: Array<string> = process.argv.slice(2);
```

Comment:
I _personally_ prefer configuration via environment rather than via CLI arguments.  As mentioned above though, I have no say in anything anymore so do what feels most right to you.  I do recommend that if we are going to go the route of not using environment variable configuration, then we should go all-out and move all configuration to CLI rather than having a mix of CLI argument and environment variable configuration.

Reply:
The idea is that we need *some* tool that has a reasonable human interface. Configuration of the guts via environment is fine, but we've shown that the human interface for that sucks and is confusing / hard to discover.